src/main/java/coffee/order/Order.java

```diff
@@ -0,0 +1,26 @@
+package coffee.order;
+
+import java.util.logging.Logger;
+
+public class Order {
+    private static final Logger logger = Logger.getLogger(Order.class.getName());
+    private int number;
+    private String name;
+
+    public Order(int number, String name) {
+        this.number = number;
+        this.name = name;
+
+        logger.info("Order " + number + " created for " + name);
```

Comment:
Logs has specific recommended format like
`logger.info("Order {} created for {}",number, name);`